Proteomics/tutorials/labelfree_vs_labelled.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: tutorial_hands_on
```

Comment:
Is it really a tutorial with hands-on? You do not really have a practical part: no hands-on where the trainers can practice.